.travis.yml

```diff
@@ -5,42 +5,30 @@ node_js:
 
 services:
   - couchdb
+  - docker
 
 git:
   depth: 30
 
-# sudo: false
+sudo:
+  false
 
-# cache:
-#   directories:
-#     - node_modules
-#     - /tmp/phantomjs
+cache:
+  directories:
+    - node_modules
+    - /tmp/phantomjs
 
 addons:
   firefox: "41.0.1"
-  apt:
-    sources:
-      - ubuntu-toolchain-r-test
-    packages:
-      - build-essential
-      - ca-certificates
-      - curl
-      - erlang-dev
-      - erlang-nox
-      - git
-      - libicu-dev
-      - libmozjs185-dev
-      - python
-      - g++
-      - g++-4.8
+
+before_install:
+  - docker pull klaemo/couchdb:2.0-dev
+  - docker run -d -p 15984:5984 klaemo/couchdb:2.0-dev --with-haproxy --with-admin-party-please -n 1
 
```

Comment:
`docker run -d -p 15984:15984 klaemo/couchdb:2.0-dev --with-admin-party-please -n 1` would also work. That leaves haproxy out of the equation ;)